### ISSUE TITLE
[Issue] export 'extend' (imported as 'Vue') was not found in 'vue' ?

### ISSUE THREAD

**Issue by jjhuang-github** [vuejs/vue-test-utils-next#765]
I'm trying to run unit test with typescript vue, and have results in the following error:

> Development environment -  "vue": "^3.1.4", "@vue/test-utils": "^2.0.0-rc.10"

> project: [ts-vue3.zip](https://github.com/vuejs/vue-test-utils-next/files/6830944/ts-vue3.zip)

**How to solve the following warning message ?**

![issue](https://user-images.githubusercontent.com/60864712/125955870-1f68ec63-6a75-4741-9238-0e12d2bebec4.PNG)

component - HelloWorld.vue
```vue
<template>
  <div class="hello">
    <h1>{{ msg }}</h1>
  </div>
</template>

<script lang="ts">
import { Options, Vue } from 'vue-class-component';

@Options({
  props: {
    msg: String
  }
})
export default class HelloWorld extends Vue {
  msg!: string
}
</script>
```

unit test 
```ts
import { expect } from 'chai'
import { shallowMount } from '@vue/test-utils'
import HelloWorld from '@/components/HelloWorld.vue'

describe('HelloWorld.vue', () => {
  it('renders props.msg when passed', () => {
    const msg = 'new message'
    const wrapper = shallowMount(HelloWorld, {
      props: { msg }
    })
    expect(wrapper.text()).to.include(msg)
  })
})
```

package.json
```json
{
  "name": "ts-vue3",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "test:unit": "vue-cli-service test:unit"
  },
  "dependencies": {
    "vue": "^3.1.4",
    "vue-class-component": "^8.0.0-rc.1",
    "vue-property-decorator": "^10.0.0-rc.2",
    "vue-router": "^4.0.0-0",
    "vuex": "^4.0.0-0",
    "vuex-class": "^0.3.2"
  },
  "devDependencies": {
    "@types/chai": "^4.2.11",
    "@types/mocha": "^5.2.4",
    "@vue/cli-plugin-router": "~4.5.0",
    "@vue/cli-plugin-typescript": "~4.5.0",
    "@vue/cli-plugin-unit-mocha": "~4.5.0",
    "@vue/cli-plugin-vuex": "~4.5.0",
    "@vue/cli-service": "~4.5.0",
    "@vue/compiler-sfc": "^3.0.0",
    "@vue/test-utils": "^2.0.0-rc.10",
    "chai": "^4.1.2",
    "node-sass": "^4.12.0",
    "sass-loader": "^8.0.2",
    "typescript": "~4.1.5"
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not dead"
  ]
}
```

**Comment by lmiller1990** [comment]
I will take a look at your reproduction this week, but we definitely have tests around class component: https://github.com/vuejs/vue-test-utils-next/blob/master/tests/components/ClassComponent.vue so I suspect a config/dependencies issue.

**Comment by xanf** [comment]
@jjhuang-github thank you for opening this one!
Actually the reason of message is bundler being too smart 
I've opened https://github.com/vuejs/vue-test-utils-next/pull/824 to fix this

**Comment by xanf** [comment]
@lmiller1990 @afontcu This could be closed, as #824 was merged
